subprojects/language-native/src/main/java/org/gradle/language/swift/tasks/CreateSwiftBundle.java

```diff
@@ -63,35 +68,42 @@ public void execute(CopySpec copySpec) {
                     }
                 });
 
-                copySpec.from(getInformationFile(), new Action<CopySpec>() {
-                    @Override
-                    public void execute(CopySpec copySpec) {
-                        copySpec.into("Contents");
-                    }
-                });
-
                 copySpec.into(getOutputDir());
             }
         });
 
+        File inputFile = getInformationFileIfExists();
+        File outputFile = new File(getOutputDir().getAsFile().get(), "Contents/Info.plist");
+        outputFile.getParentFile().mkdirs();
+        if (inputFile == null) {
+            Files.write("<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n"
+                + "<!DOCTYPE plist PUBLIC \"-//Apple//DTD PLIST 1.0//EN\" \"http://www.apple.com/DTDs/PropertyList-1.0.dtd\">\n"
+                + "<plist version=\"1.0\">\n"
+                + "<dict/>\n"
+                + "</plist>", outputFile, Charset.defaultCharset());
```

Comment:
I think it's probably not an obvious issue because this doesn't have any odd characters in it, but the XML is specifying UTF-8, but we're sometimes using something else here (e.g., whatever the default is on Windows).

Reply:
Good point, I changed it to be explicit on the encoding.